src/main/kotlin/blackjack/ui/ResultView.kt

```diff
@@ -35,6 +36,11 @@ class ResultView {
         if (gamer.getGamerType() == GamerType.DEALER) print("${gamer.getName()} 카드: ")
     }
 
+    private fun getDealerFlutteringCards(cards: List<Card>): String {
+        val (shape, number) = cards[0]
```

Comment:
destructuring declaration 활용 👍

Reply:
```suggestion
        val (shape, number) = cards.first()
```
숫자 0대신 first를 사용하시면 가독성을 챙기실 수 있습니다. 😄